modules/govuk_postgresql/manifests/backup.pp

```diff
@@ -1,11 +1,42 @@
-# FIXME: This class needs better documentation as per https://docs.puppetlabs.com/guides/style_guide.html#puppet-doc
-class govuk_postgresql::backup {
+# Class: govuk_postgresql::backup
+#
+# This class sets up local backups for PostgreSQL.
+#
+# === Parameters
+#
+# [*$auto_postgresql_backup_hour*]
+#  The hour(s) that auto_postgresql_backup(s) run(s).
+#
+# [*$auto_postgresql_backup_minute*]
+#  The minute(s) that auto_postgresql_backup(s) run(s).
+#
+# === Variables
+#
+# [*threshold_secs*]
+#  The time period in which an alert should be raised if no passive check
+#  submissions have been received.
+#
+# [*service_desc*]
+#  A simple description of the check that is alerting.
+#
+class govuk_postgresql::backup (
+  $auto_postgresql_backup_hour = '7',
+  $auto_postgresql_backup_minute = '30'
+) {
+
     $threshold_secs = 28 * 3600
     $service_desc = 'AutoPostgreSQL backup'
 
-    file {'/etc/cron.daily/autopostgresqlbackup':
+    file {'/etc/cron.d/autopostgresqlbackup':
```

Comment:
This should be created in `/usr/local/bin`. The Cron::Crondotdee defined type will create the file in `/etc/cron.d`, you just have to pass in the command that needs to be run.